### PR TITLE
update go version

### PR DESCRIPTION
**Pull request by lziest**

### CODE REVIEW

**Comment by lziest** [comment]
a few changes needs to happen before merging, due to test failures.

**Comment by lziest** [comment]
@kisom this is dramatically bigger than I thought, but take a look again anyways. I think I knock down all test failures.

**Comment by lziest** [comment]
staticcheck on mac os is so slow

**Comment by lziest** [comment]
@kisom ok to merge?

**Comment by kisom** [comment]
@lziest looks fine. I made a comment, but it stored it as pending.

File: signer/local/local_test.go [hunk 1]
```diff
@@ -144,74 +144,7 @@ func TestNewSignerFromFileEdgeCases(t *testing.T) {
 	}
 }
 
-// test the private method
```
Comment (lziest) [inline signer/local/local_test.go]: I think I need to modify the test a little bit, otherwise it is a bad test.

Comment (kisom) [inline signer/local/local_test.go]: (Y)

**Pull request merged**